test/sql/index/art/storage/test_art_names.test

```diff
@@ -0,0 +1,81 @@
+# name: test/sql/index/art/storage/test_art_names.test
+# description: Test names ART indexes
+# group: [storage]
+
+load __TEST_DIR__/test_art_names.db
+
+statement ok
+PRAGMA enable_verification;
+
+statement ok
+CREATE TABLE tbl (i INTEGER PRIMARY KEY, j INTEGER UNIQUE);
+
+statement ok
+INSERT INTO tbl SELECT range, range FROM range (3000);
+
+restart
+
+statement ok
+CREATE INDEX PRIMARY_tbl_0 ON tbl(i);
+
+statement ok
+CREATE INDEX UNIQUE_tbl_1 ON tbl(j);
+
+restart
+
+statement ok
+DROP INDEX UNIQUE_tbl_1;
+
+statement ok
+DROP INDEX PRIMARY_tbl_0;
```

Comment:
Looks like this actually removes the internal primary key once renamed - could you check/fix that?